### DISCUSSION TITLE
自由和开源软件历史上的里程碑：1969～2015 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#93] [Announcements]
# 自由和开源软件历史上的里程碑：1969～2015 - 「开源之道」

https://opensourceway.community/posts/opensource/milestones_in_free_and_open_source_software_history/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

可惜了, 湾区日报, 只是自己习惯的一个日报, 慢慢能变成一组复杂的系统和应用...
原点, 也只是自己的阅读习惯...

类似的 开源导报 其实也是个好点...

可以使用 substack.com 作起来;-)
